src/overlays/actors/ovl_En_G_Switch/z_en_g_switch.c

```diff
@@ -210,14 +210,14 @@ void EnGSwitch_WaitForObject(EnGSwitch* this, GlobalContext* globalCtx) {
 }
 
 void EnGSwitch_SilverRupeeTracker(EnGSwitch* this, GlobalContext* globalCtx) {
-    static s8 rupeePitches[] = { 0, 2, 4, 5, 7 };
+    static s8 majorScale[] = { 0, 2, 4, 5, 7 };
 
-    if (this->pitchIndex < sCollectedCount) {
+    if (this->noteIndex < sCollectedCount) {
         if (sCollectedCount < 5) {
             // sound?
```

Comment:
```suggestion
```

Reply:
No, this is just a translation of the japanese string. Should arguably have quotes around I think, but that's a larger change, and would need to be done consistently across the project.

Reply:
fwiw most debug strings do have quotes on them, and we usually tell people to add them in review. there are certainly a bunch that got missed though

Reply:
but yeah, you can add the quotes now if you like